packages/button/src/Button.spec.tsx

```diff
@@ -138,4 +139,14 @@ describe('packages/Button', () => {
 
     expect(buttonComponent.tagName.toLowerCase()).toBe('div');
   });
+
+  test(`renders a button with an arbitrary element passed in glyph prop`, () => {
```

Comment:
What exactly are we testing here? I don't see an expect statement... Perhaps we could do something like 

`const { getByTestId } = render( <Button glyph={<Icon glyph="Trash" />} data-testid="trash-button">My Trash Button</Button>)` and checking to make sure that the trash-button contains the Icon?

Reply:
Dave recommended that I add a test checking if something renders when the glyph prop is set to a value other than null. I don't know that I did it correctly as my jest knowledge is a little rusty. I can definitely use your suggestion instead.

Reply:
Feel free to test for that, just be sure to add an assertion statement :)

Reply:
Ok, went ahead and fixed it. Thanks for catching that!

Reply:
👍